src/Microsoft.IdentityModel.Tokens/SupportedAlgorithms.cs

```diff
@@ -104,7 +104,7 @@ internal static class SupportedAlgorithms
             SecurityAlgorithms.EcdhEsA256kw
         };
 
-#if NET461 || NET462 || NET472 || NETSTANDARD2_0 || NET6_0_OR_GREATER
+#if NET462 || NET472 || NETSTANDARD2_0 || NET6_0_OR_GREATER
```

Comment:
Anytime we see: #if NET462 || NET472 || NETSTANDARD2_0 || NET6_0_OR_GREATER, we can remove the #if.

Reply:
fixed